src/components/Note.tsx

```diff
@@ -111,6 +94,9 @@ const Note = React.memo(({ path }: { path: Path }) => {
     }
   }
 
+  /** Sets the cursor on the note's thought as it is being edited. */
+  const onFocus = () => store.dispatch(setNoteFocus({ value: true }))
```

Comment:
diff from the original fn:

setCursor({
      ~path: simplePath~, // does not simplify a path, we don't need it
      ~cursorHistoryClear: true~, // why do we need it?
      editing: true, // does it inside `setNoteFocus`
      noteFocus: true, // does it via `setNoteFocus`
    })